projects/dex-ui/src/pages/Wells.tsx

```diff
@@ -35,72 +85,163 @@ export const Wells = () => {
           </WellDetail>
         </Td>
         <Td>
-          <DataText>price function</DataText>
+          <WellPricing>{wellFunctionNames[index] ? wellFunctionNames[index] : "Price Function"}</WellPricing>
+        </Td>
+        <Td align="right">
+          <TradingFee>0.00%</TradingFee>
+        </Td>
+        <Td align="right">
+          <Amount>${wellLiquidity[index] ? wellLiquidity[index]!.toHuman("0,0.00") : "-.--"}</Amount>
```

Comment:
toHuman() no longer accepts parameters like this, only undefined or literal string "short". This should've given you a typescript error 🤔 

Please update all number displays to use `.toHuman("short")` for formatting.